### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by jmlongriver12** [AmeyaHarmalkar/therML#2]
I got an error when running "python utility/get_antiberty_likelihoods.py" under "ab-ptlm" as belows:
"AttributeError: 'IgFold' object has no attribute 'tokenizer'"

the version of igfold that I installed is 0.3.0, could you please help me find the right version? thanks

**Comment by jourmore** [comment]
Indeed, the specific operating environment is not given, and I have a similar error：No module named 'evo.tokenization'.
